Client.xcodeproj/project.pbxproj

```diff
@@ -9,6 +9,143 @@
 /* Begin PBXBuildFile section */
 		0AE491A61A41C88C0046C724 /* BackForwardListViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0AE491A51A41C88C0046C724 /* BackForwardListViewController.swift */; };
 		0B1C05D71A798B1F004C78B0 /* UIImageViewAligned.m in Sources */ = {isa = PBXBuildFile; fileRef = 0B1C05D61A798B1F004C78B0 /* UIImageViewAligned.m */; };
+		0B2B65E41B0683BE001D8481 /* .gitignore in Resources */ = {isa = PBXBuildFile; fileRef = 0B2B65301B0683BD001D8481 /* .gitignore */; };
```

Comment:
We don't want all these files in the project. They are also part of the Client target which means they are processed/copied to the packaged build. Can you remove all of them except `Readability.js`? It is fine that Carthage checks them out, we just don't want them in the project.